crates/proc-macros/src/lib.rs

```diff
@@ -35,11 +35,14 @@ use syn::{parse::Parse, parse_macro_input, Attribute, Ident, ItemTrait, LitStr,
 
 struct VersionedRpcAttr {
     version: String,
+    namespace: String,
 }
 
 impl Parse for VersionedRpcAttr {
     fn parse(input: syn::parse::ParseStream) -> syn::Result<Self> {
         let version = input.parse::<LitStr>()?.value();
+        input.parse::<syn::Token![,]>()?;
+        let namespace = input.parse::<LitStr>()?.value();
```

Comment:
Should probably add a check to make sure the namespace is not an empty string here + custom macro error message. The `indoc` crate can be quite useful for this purpose.